composer.json

```diff
@@ -11,12 +11,18 @@
     ],
     "autoload": {
         "psr-4": {
-            "Psr\\Http\\Message\\": "src"
+            "Psr\\Http\\Message\\": [
+                "src",
+                "test"
```

Comment:
You should move test directory to autoload-dev. We don't need libs tests after required.